app/views/receipts/new.html.slim

```diff
@@ -1,2 +1,2 @@
 .container.dashboard-container
-  = react_component "transactions/CreateTransaction", { artist: current_artist }
+  = react_component "transactions/TransactionForm", { artist: current_artist }
```

Comment:
You don't actually need to set up routes for the TransactionForm component. You can just import them in React and refer to them there, considering that these components are imported wherever they are used